### PR TITLE
Run URL filter lambda test using localstack on ci before deployment

### PR DESCRIPTION
**Pull request by rohandudam**
Added job to run URl filter Lambda test before staging and production jobs.
Ran job successfully using 'act' tool Locally.
act-git-action-ran

### CODE REVIEW

File: .github/workflows/url_filtering_lambda_rohini.yml [hunk 1]
```diff
@@ -5,10 +5,56 @@ on:
...
+        run: |
+          cd tests/
+          docker-compose -f docker-compose.yaml up -d
+      - name: Wait for LocalStack to be ready
+        run: |
+          timeout 300 bash -c 'until echo > /dev/tcp/localhost/4566; do sleep 1; done'
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8" # Replace with your desired Python version
```
Comment (sravantit25) [inline .github/workflows/url_filtering_lambda_rohini.yml:34]: Can you remove the comment?

File: .github/workflows/url_filtering_lambda_rohini.yml [hunk 2]
```diff
@@ -5,10 +5,56 @@ on:
...
+      - name: Install dependencies
+        run: |
+          cd tests/
+          pip install -r requirements.txt # Replace with your actual requirements file
```
Comment (sravantit25) [inline .github/workflows/url_filtering_lambda_rohini.yml:38]: Can you remove the comment.

**Review by sravantit25 — APPROVED** [review]
@rohandudam The PR looks fine. I have minor suggestions to remove the comments. I tested it by using GitHub Action. I created a branch out of yours by using the name sprint in the name and made a small docstring change in one of the files. It triggered the flow, the run was successful.

File: .github/workflows/url_filtering_lambda_rohini.yml (hunk 1 above)
Comment (rohandudam) [reply .github/workflows/url_filtering_lambda_rohini.yml]: done

File: .github/workflows/url_filtering_lambda_rohini.yml (hunk 2 above)
Comment (rohandudam) [reply .github/workflows/url_filtering_lambda_rohini.yml]: done

**Pull request merged**